pkg/manager-server/file_system_gfs2.go

```diff
@@ -102,21 +102,20 @@ func (f *FileSystemGfs2) Mount(mountpoint string) error {
 		return err
 	}
 
-	if _, err := f.run(fmt.Sprintf("mount %s %s", f.FileSystemLvm.devPath(), mountpoint)); err != nil {
+	mounted, err := f.IsMounted(mountpoint)
```

Comment:
If this can happen, should you also check if the directory has been created above on line 101?

Reply:
Take a look at file_system_lustre:Mount() to see how I handled the mkdir, and allowed for it to already exist.

Reply:
-p option
> no error if existing, make parent directories as needed

Reply:
@roehrich-hpe MkdirAll says "If path is already a directory, MkdirAll does nothing and returns nil" - so why the `os.IsExist` check?

Reply:
Expand the example here: https://pkg.go.dev/os#MkdirAll

That's probably where I got it.

Reply:
I do think we should not fork+exec to do a mkdir.  We should use the Golang library instead.

Reply:
> Expand the example here: https://pkg.go.dev/os#MkdirAll
Wow, even their example is wrong

> I do think we should not fork+exec to do a mkdir. We should use the Golang library instead.
agreed - on it